backend/src/model/dto/map_impl.rs

```diff
@@ -21,3 +21,21 @@ impl From<Map> for MapDto {
         }
     }
 }
+
+impl<T> From<(Map, T)> for MapDto {
```

Comment:
Please add a short comment describing what this does (for Rust-novices)

Reply:
I simplified it. It think it should be clear enough with that. Apart from that it is a simple from impl like everywhere else as well, just for a tuple with generics.